WordPress/Classes/Services/CommentServiceRemoteCoreRESTAPI.swift

```diff
@@ -146,10 +151,13 @@ private extension RemoteComment {
         self.status = comment.status.commentStatusType?.description
         self.type = comment.commentType.type
 
+        if let ext = try? comment.additionalFields.parseWpcomCommentsExtension() {
+            self.postTitle = ext.post?.title
+            self.isLiked = ext.iLike
+            self.likeCount = NSNumber(value: ext.likeCount)
```

Comment:
These fields have been added to the WP.com endpoints, so we can parse and use them now.